third_party/seal.BUILD

```diff
@@ -0,0 +1,29 @@
+load("@rules_foreign_cc//tools/build_defs:cmake.bzl", "cmake_external")
+
+filegroup(
+    name = "src", 
+    srcs = glob(["**"]), 
+    visibility = ["//visibility:public"]
+)
+
+cmake_external(
+   name = "seal",
+   cmake_options = [
+        "-DSEAL_USE_CXX17=17",
+        "-DSEAL_USE_INTRIN=ON",
+        "-DSEAL_USE_MSGSL=OFF",
+        "-DSEAL_USE_ZLIB=OFF",
```

Comment:
Are we sure we want to disable ZLIB? I've seen some updates to the tests regarding this, while the Cmake build is using ZLIB, we should still be testing this, so how about enabling it here and keep the tests? I'm actually unaware of advantages of disabling it so would love to know more about the different cases

Reply:
Thank for noticing this, I forgot about it. I will fix it.